### PR TITLE
Closes #3387: Docs for SDK Metrics on Ansible Operators

### LINKED ISSUE #3387: Provide Metrics list
Provide documented list of standard metrics emitted by operator SDK.

### PR DESCRIPTION
**Pull request by VenkatRamaraju**
**Description of the change:**
Added docs for the set of metrics that are internally collected by SDK for Ansible operators.

**Motivation for the change:**
Close #3387.

### CODE REVIEW

File: website/content/en/docs/building-operators/ansible/reference/internal_metrics.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+The Ansible Operator comes with three internal metrics that provide an insight to the frequency and time of operator reconciliations. These metrics can be
+scraped by a Prometheus instance or any other openmetrics system. To publish operator metrics and scrape them with an openmetrics system such as Prometheus, view 
+[Kubebuilder documentation](https://book.kubebuilder.io/reference/metrics.html) on publishing metrics.
+
+The default metrics recorded in Operator-SDK are collected in a [histogram](https://prometheus.io/docs/practices/histograms/).
```
Comment (jmrodri) [inline website/content/en/docs/building-operators/ansible/reference/internal_metrics.md:5]: When referring to the command you use `operator-sdk` when referring to the project name use "Operator SDK"

```suggestion
The default metrics recorded in Operator SDK are collected in a [histogram](https://prometheus.io/docs/practices/histograms/).
```

**Review by jmrodri — APPROVED** [review]
very good. Just one minor issue. Once that's fixed you can merge this. Approving conditionally

**Review by varshaprasad96 — APPROVED** [review]
/lgtm

**Pull request merged**